dotnet/Simple.Sauce/SauceOptions.cs

```diff
@@ -54,6 +72,24 @@ public void WithChrome(string chromeVersion)
             };
         }
 
+        public IWebDriver CreateEdgeBrowser()
```

Comment:
Where's the Firefox case?

Reply:
To expound on this, we shouldn't need anything that is specific to a browser here. For Typing we should be able to use the superclass.